rapids-metadata.json

```diff
@@ -395,7 +395,7 @@
             "libwholegraph": {
               "has_conda_package": true,
               "has_cuda_suffix": true,
-              "has_wheel_package": false,
+              "has_wheel_package": true,
```

Comment:
This is changing the metadata for the RAPIDS 24.08 release, which is not what we want.

This file is automatically generated:

https://github.com/rapidsai/rapids-metadata/blob/8b2485a4acd1e3aa3a65a05f25d123cd84ab8b38/.pre-commit-config.yaml#L49-L64

Running `pre-commit run --all-files` should correctly update it.